### ISSUE TITLE
Have you ever considered exporting the rtmp protocol as an independent golang library?

### ISSUE THREAD

**Issue by Akagi201** [ossrs/httpx-static#61]
I dive into the protocol directory.

I found that the protocol sub-package only depends on `core/utility.go` and `core.Context`.

I think you can create a util sub-package and put `core/utility.go` and `core.Context` in it.

What do you think of it? Or I can do it myself, and make a PR.

Append...

The protocol sub package also depends  on `config.go` and `logger.go`. It may be not easy to export the protocol package.

**Comment by winlinvip** [comment]
Good idea, we should export the protocol to https://github.com/ossrs/go-oryx-lib. Please file a PR~
